cypress/commons/actions/generic/TableParameters.js

```diff
@@ -216,6 +232,8 @@ export const TableParameters = {
   exportFile,
   exportCSV,
   exportXLSX,
+  addRow,
+  deleteRows,
```

Comment:
Can you also export `getAddRowButtonInput` and `getDeleteRowsButtonInput` too please? It can be useful if we want to test whether they are visible, enabled, or if their tooltips work as expected.

Reply:
DONE